src/routes/customer.js

```diff
@@ -1,15 +1,124 @@
 const express = require('express');
-
 const router = express.Router();
 
 const customerController = require('../controllers/customerController');
 const { validCustomer } = require('../validation/validator');
 
+/**
+ * @swagger
+ * tags:
+ *   name: Customer
+ *   description: API endpoints for customer
+ */
+
+/**
+ * @swagger
+ * /customer:
+ *   get:
+ *     tags: [Customer]
+ *     summary: Get all customers
+ *     responses:
+ *       200:
+ *         description: Successful operation
+ *       500:
+ *         description: Internal server error
+ */
 router.get('/', customerController.getAllCustomers);
+
+/**
+ * @swagger
+ * /customer:
+ *   post:
+ *     tags: [Customer]
+ *     summary: Add a new customer
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/Customer'
```

Comment:
You should create a schema for the customer in the configuration/swagger.js. look at the swagger.js you will find out